src/pages/nahyunlee/Login/login-nahyun.js

```diff
@@ -1,36 +1,84 @@
 import React from 'react';
 //import {Link} from 'react-router-dom';
-import { withRouter } from 'react-router-dom';
-import './Login.scss';
+import { withRouter, Link } from 'react-router-dom';
+import './LoginNa.scss';
 
 class LoginNa extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      idValue: '',
+      pwValue: '',
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      idValue: `${e.target.value}`,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pwValue: `${e.target.value}`,
+    });
+  };
```

Comment:
```suggestion
  handleInputs = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```
- 이런 식으로 함수를 간단하게 하나로 줄일 수 있습니다!
- 단, input 태그에 name 속성을 부여해 주셔야 합니다!

Reply:
넵! input 태그에 name 속성 부여하여 수정완료하였습니다. 감사합니다.